docs/book/src/multiversion-tutorial/api-changes.md

```diff
@@ -52,4 +52,4 @@ Now, let's copy over our existing types, and make the change:
 
 Now that we've got our types in place, we'll need to set up conversion...
 
-[cronjob-sched-code]: /TODO.md
+[cronjob-sched-code]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/multiversion-tutorial/testdata/project/api/v2/cronjob_types.go "ConJob Code"
```

Comment:
We must use relative PATHS?
Can you please fix this one?

Reply:
Fixed in new commit